public/docs/ts/latest/tutorial/toh-pt5.jade

```diff
@@ -83,57 +105,55 @@ code-example(language="bash").
   * `AppComponent` class to `HeroesComponent`
   * Selector `my-app` to `my-heroes`
 
-:marked
-+makeExample('toh-5/ts/app/heroes.component.ts', 'heroes-component-renaming', 'app/heroes.component.ts (showing renamings only)')(format=".")
++makeExcerpt('app/heroes.component.ts (showing renamings only)', 'renaming')
 
 :marked
   ## Create *AppComponent*
+
   The new `AppComponent` will be the application shell.
   It will have some navigation links at the top and a display area below for the pages we navigate to.
 
   The initial steps are:
 
-  * create a new file named `app.component.ts`.
-  * define an `AppComponent` class.
-  * `export` it so we can reference it during bootstrapping in `app.module.ts`.
-  * expose an application `title` property.
-  * add the `@Component` metadata decorator above the class with a `my-app` selector.
-  * add a template with `<h1>` tags surrounding a binding to the `title` property.
-  * add the `<my-heroes>` tags to the template so we still see the heroes.
-  * add the `HeroesComponent` to the root NgModule's `declarations` array so Angular recognizes the `<my-heroes>` tags.
-  * add the `HeroService` to the root NgModule's `providers` array because we'll need it in every other view.
-  * add the supporting `import` statements.
+  * Create the file <span ngio-ex>app/app.component.ts</span>.
+  * Define an <span if-docs="ts">exported</span> `AppComponent` class.
+  * Add an `@Component` !{_decorator} above the class with a `my-app` selector.
+  * Move from `HeroesComponent` to `AppComponent` the
+    * `title` class property, and
+    * `@Component` template `<h1>` element containing a `title` binding.
+  * Add a `<my-heroes>` element to the app template just below the heading so we still see the heroes.
+  * Add `HeroesComponent` to the `!{_declsVsDirectives}` !{_array} of `!{_AppModuleVsAppComp}` so Angular recognizes the `<my-heroes>` tags.
+  * Add `HeroService` to the  `providers` !{_array} of `!{_AppModuleVsAppComp}` because we'll need it in every other view.
+  * Remove `HeroService` from the `HeroesComponent` `providers` !{_array} since it has been promoted.
+  * Add the supporting `import` statements for `AppComponent`.
 
   Our first draft looks like this:
-+makeTabs(
-  `toh-5/ts/app/app.component.1.ts,
-  toh-5/ts/app/app.module.1.ts`,
-  ',',
-  `app/app.component.ts (v1),
-  app/app.module.ts (v1)`)
 
-:marked
-.callout.is-critical
-  header Remove <i>HeroService</i> from the <i>HeroesComponent</i> providers
-  :marked
-    Go back to the `HeroesComponent` and **remove the `HeroService`** from its `providers` array.
-    We are *promoting* this service from the `HeroesComponent` to the root `NgModule`.
-    We ***do not want two copies*** of this service at two different levels of our app.
+block app-comp-v1
+  +makeTabs(
+    `toh-5/ts/app/app.component.1.ts,
+    toh-5/ts/app/app.module.1.ts`,
+    ',',
+    `app/app.component.ts (v1),
```

Comment:
In the Dart version, I see the title `lib/app_component_1.dart`.

(I didn't expect to see the `_1`, and it doesn't appear in the TS version.)